src/test/java/HomePage.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    By avatarIcon = By.cssSelector("img.avatar");
+    By firstPlaylist = By.cssSelector("#playlists > ul > li:nth-child(3) > a");
+    By editSelection = By.cssSelector("#playlists > ul > li:nth-child(3) > nav > ul > li:nth-child(1)");
+    By nameField = By.cssSelector("[name='name']");
+    By verification = By.cssSelector("div.show.success");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```